src/test/java/org/dcsa/bkg/controller/BKGEventControllerTest.java

```diff
@@ -0,0 +1,273 @@
+package org.dcsa.bkg.controller;
```

Comment:
IS the idea with just "just" to get started? We are missing test cases for all positive results...

Reply:
There are positive test cases as well in this.

Reply:
Do you mean for each request param as well?

Reply:
That would be my expectation. If for some reason the code is removed so a param is no longer valid - how will our tests catch this?